src/south/SQLDbToFile/SQLDbToFile.class.js

```diff
@@ -93,6 +93,26 @@ class SQLDbToFile extends ProtocolHandler {
     }
   }
 
+  /**
+   * Function used to parse an entry and update the lastCompletedAt if needed
+   * @param {*} entry - on sql result item
+   * @return {void}
+   */
+  checkLastCompletedAt(entry) {
+    if (entry[this.timeColumn] && (entry[this.timeColumn] instanceof Date) && (entry[this.timeColumn] > new Date(this.lastCompletedAt))) {
+      this.lastCompletedAt = entry[this.timeColumn].toISOString()
+    } else if (entry[this.timeColumn]) {
```

Comment:
If I read correctly, nothing happens if entry[this.timeColumn] is undefined or null should we do a silly log?

Reply:
I don't think we should: this function is called for every entry of a query results. If it's a big query, the logger may overload the logs. Even if it's a silly mode, I'd prefer not to.